### PR TITLE
More NaN usage and create_model cleanup

### PR DESCRIPTION
**Pull request by karlb**
This PR is a grab bag of smaller cleanups, so I recommend to review commit by commit.

### CODE REVIEW

File: causing/estimate.py [hunk 1]
```diff
@@ -20,6 +20,7 @@ def sse_orig_alg(direct, model_dat):
         direct = np.array(direct).reshape(-1)
         mx, my = utils.directmat_alg(direct, model_dat["idx"], model_dat["idy"])
         ex_hat, _ = utils.total_effects_alg(mx, my, model_dat["edx"], model_dat["edy"])
+        ex_hat = utils.nan_to_zero(ex_hat)
```
Comment (Holger-Bartel) [inline causing/estimate.py:23]: You changed ex_hat foor good not just temporarily. Was that really intended?

File: output/example/graphs.json [hunk 2]
```diff
@@ -54,23 +54,23 @@
     ],
     "ex_hat_std": [
         [
-            0.479269,
+            NaN,
             0.0
         ],
         [
-            5.86522,
-            0.802755
```
Comment (Holger-Bartel) [inline output/example/graphs.json]: those should not be replaced

File: output/example/graphs.json [hunk 3]
```diff
@@ -54,23 +54,23 @@
...
+            NaN,
+            NaN
         ],
         [
-            0.221734,
-            0.199683
```
Comment (Holger-Bartel) [inline output/example/graphs.json]: those should not be replaced

File: output/example/graphs.json [hunk 4]
```diff
@@ -129,8 +129,8 @@
         1.05478
     ],
     "exj_hat_std": [
-        0.221734,
-        0.199683
```
Comment (Holger-Bartel) [inline output/example/graphs.json]: those should not be replaced

File: output/example/graphs.json [hunk 5]
```diff
@@ -566,27 +566,27 @@
...
         ],
         [
-            5.95873,
+            NaN,
             0.0,
             0.0
         ],
         [
-            6.1991,
-            0.746567,
```
Comment (Holger-Bartel) [inline output/example/graphs.json]: those should not be replaced

File: output/example/graphs.json [hunk 6]
```diff
@@ -652,8 +652,8 @@
         1.0
     ],
     "eyj_hat_std": [
-        6.1991,
-        0.746567,
```
Comment (Holger-Bartel) [inline output/example/graphs.json]: those should not be replaced

File: output/example/graphs.json [hunk 7]
```diff
@@ -1285,16 +1285,16 @@
...
     "eyx_hat_std": [
         [
-            0.221734,
-            0.0
+            NaN,
+            NaN
         ],
         [
-            0.0,
-            0.199683
```
Comment (Holger-Bartel) [inline output/example/graphs.json]: those should not be replaced

File: output/example/graphs.json [hunk 8]
```diff
@@ -1374,18 +1374,18 @@
...
         [
-            0.0,
-            0.0,
+            NaN,
+            NaN,
             NaN
         ],
         [
-            5.71979,
-            0.0,
```
Comment (Holger-Bartel) [inline output/example/graphs.json]: those should not be replaced

File: output/example/graphs.json [hunk 9]
```diff
@@ -1374,18 +1374,18 @@
...
+            NaN,
+            NaN,
             NaN
         ],
         [
-            0.47931,
-            0.746567,
```
Comment (Holger-Bartel) [inline output/example/graphs.json]: those should not be replaced

File: output/example/graphs.json [hunk 10]
```diff
@@ -1537,12 +1537,12 @@
     ],
     "mx_hat_std": [
         [
-            0.479269,
+            NaN,
             0.0
         ],
         [
             0.0,
-            0.802755
```
Comment (Holger-Bartel) [inline output/example/graphs.json]: those should not be replaced

**Review by Holger-Bartel — CHANGES_REQUESTED** [review]

File: causing/estimate.py (hunk 1 above)
Comment (karlb) [reply causing/estimate.py:23]: The modified ex_hat does not leave the function, so that is fine.

File: output/example/graphs.json (hunk 2 above)
Comment (karlb) [reply output/example/graphs.json]: Well spotted! I missed them among all the good changes. This is fixed now along with the other unwanted NaNs.

**Review by Holger-Bartel — APPROVED** [review]

**Pull request merged**